### PR TITLE
More

### PR DESCRIPTION
**Pull request by tompec**

### CODE REVIEW

File: wildcard.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "cko.kr",
   "cloudns.cc",
   "cloudns.cx",
+  "co.kr",
```
Comment (danhstevens) [inline wildcard.json]: `co.kr` is a TLD and includes Korean versions of sites like Google, eBay, etc.

**Comment by tompec** [comment]
Oops, sorry, my bad. Fixed.

**Comment by danhstevens** [comment]
No worries. Looks like `mooo.com` and `ignorelist.com` are "Free DNS" services - these _could_ have some legit mail services (ex: personal mail servers) so blocking them as a wildcard might be a bit overkill as well. Perhaps the criteria should be if the domain is owned/operated by a disposable service (and they have several subdomains) then they can be added to the wildcard.

**Comment by tompec** [comment]
Yup, that's what we said.

Here are the subdomains:

```
xnmail.mooo.com
cousinit.mooo.com
ramjane.mooo.com
mailinatorzz.mooo.com
icfu.mooo.com
kmail.mooo.com
3675.mooo.com
haniv.ignorelist.com
mustbe.ignorelist.com
```

**Comment by tompec** [comment]
@danhstevens Can you explain why you removed `mooo.com` and `ignorelist.com`, since I thought that a domain was considered wildcard as soon as it had more than 1 subdomain used as disposable service.
Ref: https://github.com/ivolo/disposable-email-domains/pull/108#issuecomment-291268818
Thanks!

**Comment by tompec** [comment]
@danhstevens Gotcha. Makes sense and I'm ok with this, but I think there's already a few free domain providers that are listed in the wildcard list so we should delete them.
I'll make a PR.

**Comment by danhstevens** [comment]
@tompec Sounds great and **many** thanks for all your help maintaining this repo!

**Pull request merged**